t/20_sdaf_deployment_library.t

```diff
@@ -494,4 +473,28 @@ subtest '[playbook_settings] Verify playbook order' => sub {
     is $playbook_list[10], 'playbook_05_03_sap_app_install.yaml', 'Playbook #11 must be: playbook_05_03_sap_app_install.yaml';
 };
 
+subtest '[ansible_show_status] HanaSR status commands' => sub {
+    my $ms_sdaf = Test::MockModule->new('sles4sap::sap_deployment_automation_framework::deployment', no_auto => 1);
+    my @commands;
+    $ms_sdaf->redefine(record_info => sub { return });
```

Comment:
nice to have it redefined with `note`

Reply:
Not the cleanest one, but I changed it so it shows something useful: 
```
    $ms_sdaf->redefine(record_info => sub { note(join(' ', 'RECORD_INFO -->', $_[0], ':', $_[1])); });
    $ms_sdaf->redefine(ansible_execute_command => sub { push @commands, @_; return $_[1]});
```

```
    # RECORD_INFO --> OS info : cat /etc/os-release
    # RECORD_INFO --> DB processes : ps -ef | grep hdb
    # RECORD_INFO --> DB cluster : sudo crm status full
    # RECORD_INFO --> HanaSR status : sudo SAPHanaSR-showAttr
    # RECORD_INFO --> ENSA2 cluster : sudo crm status full
    # RECORD_INFO --> NW PAS : ps -ef | grep sap

```